### PR TITLE
Indents to spaces (it was mixed). Maven Javadoc Plugin has been configured

### PR DESCRIPTION
**Pull request by recena**
- [x] `pom.xml` file used a mixed tab format (tabs and spaces
- [x] Configure Maven Javadoc Plugin to release using JDK8

@reviewbybees

### CODE REVIEW

**Comment by tfennelly** [comment]
:bee:

File: pom.xml [hunk 1]
```diff
@@ -1,110 +1,110 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>org.jenkins-ci.plugins</groupId>
-		<artifactId>plugin</artifactId>
-		<version>1.554.2</version>
```
Comment (amuniz) [inline pom.xml]: Was it needed?

File: pom.xml [hunk 2]
```diff
@@ -1,110 +1,110 @@
...
-		<license>
-			<name>MIT License</name>
-			<url>http://www.opensource.org/licenses/mit-license.php</url>
-		</license>
-	</licenses>
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>1.554.3</version>
```
Comment (amuniz) [inline pom.xml]: Was it needed?

File: pom.xml [hunk 3]
```diff
@@ -296,57 +300,57 @@
...
+                <!--This plugin's configuration is used to store Eclipse m2e settings
+                    only. It has no influence on the Maven build itself. -->
+                <plugin>
+                    <groupId>org.eclipse.m2e</groupId>
+                    <artifactId>lifecycle-mapping</artifactId>
+                    <version>1.0.0</version>
+                    <configuration>
+                        <lifecycleMappingMetadata>
+                            <pluginExecutions>
+                                <!--<pluginExecution>
```
Comment (amuniz) [inline pom.xml]: Since you are cleaning things, why not remove this commented lines?

**Comment by amuniz** [comment]
:bee: (only minor comments)

File: pom.xml (hunk 2 above)
Comment (recena) [inline pom.xml]: @amuniz No, it was not but 1.554.3 provides fixes interesting. it was a way to promote the last LTS on 1.554

File: pom.xml (hunk 3 above)
Comment (recena) [inline pom.xml]: @amuniz Indeed but I tried to do the minimal changes. Maybe in another PR.

**Comment by oleg-nenashev** [comment]
:bee: in any case

**Comment by recena** [comment]
@reviewbybees done

**Comment by recena** [comment]
@amuniz @tfennelly @oleg-nenashev Thanks so much.

**Pull request merged**